src/lib/core/crash.c

```diff
@@ -382,7 +382,7 @@ crash_report_feedback_daemon(struct crash_info *cinfo)
 	snprintf_safe("}");
 	snprintf_safe("}");
 
-	pr_debug("crash dump: %s", head);
+	pr_syserr("crash dump: %s", head);
```

Comment:
This isn't the place where the stack trace is printed on SIGSEGV. It's printed here:

https://github.com/tarantool/tarantool/blob/6947ed76b5bcaef1e590f71cc8e206dcd300e6d4/src/lib/core/crash.c#L517

It's always printed to `stderr`, not to the log, irrespective of the log level. That is, if Tarantool crashes, you won't find the crash dump in the Tarantool log. You'll find it in the system log (`journalctl`).

You patched the feedback daemon code instead - there the stack trace is printed solely for debug purposes, so usage of `pr_debug` is correct.

It's always been this way, probably because the log may be not configured when Tarantool crashes or the bug may actually be in the logger.

Reply:
I changed log message ("crash dump" -> "crash dump for feedback daemon") to avoid such confusion.

Reply:
Probably, we should expand it more (something like "crash dump for feedback daemon, not for crash investigation"), but such a message looks silly.